app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,71 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('usr', 'share'),
+  path.join('usr', 'share', 'ubuntu'),
+  path.join('usr', 'share', 'gnome'),
+  path.join('usr', 'local', 'share'),
+  path.join('var', 'lib', 'snapd', 'desktop')
```

Comment:
Do you have set up environment? If so you can just run application by `npm run dev` and add console.log/debug in this file and check what is inside yours appDirs/DIRECTORIES